src/components/try-hcp-callout/components/description/index.tsx

```diff
@@ -0,0 +1,14 @@
+import mitigateWidows from '@hashicorp/platform-util/text/mitigate-widows'
+import s from './description.module.css'
+
+/**
+ * Description component, for use in HCP callout cards.
+ */
+export function Description({ description }: { description: string }) {
```

Comment:
There are a couple of "variants" of this component already. With this in mind, I've broken things down into pretty small pieces, so that the small pieces can be composed to create "variants".

Reply:
Could these building blocks be eventually shared between all callouts? Seeing that we have a few similar callout variants already. That may already be what you are describing. Just future thinking 💭

Reply:
Here are the callouts I see in the repo currently: 
- interactive lab
- callout card
- mdx hcp callout 
- branded callout (on the tutorials product pages)

Might be cool to create a shared abstraction if the designs are super similar (or in come cases the same minus the background assets). Not necessary for this PR though.

Reply:
I dig it - added a task for the future: 🎟 [Assess abstracting atomic "callout" components](https://app.asana.com/0/1202097197789424/1203396252179033/f)